### ISSUE TITLE
I started syncing and then closed sketch document.

### ISSUE THREAD

**Issue by sokos** [avocode/avocode#1570]
IMPORTANT: Please try to describe how did it happen.

I started syncing and then closed sketch document.

-----
App v. `2.14.8`
Avocode UID: 50887
```
Error: Api#updateUploadSlot()() -> error (status: 400, error: {"__all__":["Parent slot must be uploaded before child slots!"]})
    at Object.normalizeError (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:16897)
    at Object.normalizeResponse (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:16733)
    at t.normalizeResponse (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2003680)
    at file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2000698
    at Request._callback (file:///Applications/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2002521)
    at Request.self.callback (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:186:22)
    at emitTwo (events.js:111:20)
    at Request.emit (events.js:191:7)
    at Request.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:1081:10)
    at emitOne (events.js:96:13)
    at Request.emit (events.js:188:7)
    at IncomingMessage.<anonymous> (/Applications/Avocode.app/Contents/Resources/app.asar/node_modules/request/request.js:1001:12)
    at Object.onceWrapper (events.js:290:19)
    at emitNone (events.js:91:20)
    at IncomingMessage.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by jakubzitny** [comment]
Hi @sokos, thanks for reporting this bug. I think if you try uploading again, you should not have such problems, this should hapen very rarely. We'll try to fi it as soon as possible though.

**Comment by bolekkerous** [comment]
Hey, @sokos  -
could you please try to log out and log in again? It seemed to help in a similar issue here: #1611

**Comment by jakubzitny** [comment]
Hi there, this should be fixed in latest Avocode `2.15.0`. You can download it from [avocode.com/download](https://avocode.com/download) or use auto-update to install it. Please give it a try and let us know if you have any more issues.
